qiita_db/handlers/reference.py

```diff
@@ -6,7 +6,7 @@
 # The full license is in the file LICENSE, distributed with this software.
 # -----------------------------------------------------------------------------
 
-from tornado.web import RequestHandler
+from .oauth2 import OauthBaseHandler, authenticate_oauth
 
```

Comment:
It is really confusing that it breaks pep8 - as that is blank line is
specific defined by pep8. is it because you are using relative imports? If
so change to global.

2015-12-21 10:33 GMT-08:00 Joshua Shorenstein <notifications@github.com>:

> In qiita_db/handlers/reference.py
> <https://github.com/biocore/qiita/pull/1562#discussion_r48175040>:
>
> > @@ -6,7 +6,7 @@
> >  # The full license is in the file LICENSE, distributed with this software.
> >  # -----------------------------------------------------------------------------
> >
> > -from tornado.web import RequestHandler
> > +from .oauth2 import OauthBaseHandler, authenticate_oauth
> >
>
> breaks pep8, can't.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/biocore/qiita/pull/1562/files#r48175040>.
>



-- 
Jose Navas